<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseDataWriterWdg</class>
 <widget class="QWidget" name="BaseDataWriterWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutBool">
     <item>
      <widget class="XMatLabel" name="lbBool">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>布尔量:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbBool"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutInt">
     <item>
      <widget class="XMatLabel" name="lbInt">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>整数值:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letInt"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutReal">
     <item>
      <widget class="XMatLabel" name="lbReal">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>浮点数:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letReal"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutString">
     <item>
      <widget class="XMatLabel" name="lbString">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>字符串:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatLineEdit" name="letString"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XMatLineEdit</class>
   <extends>QLineEdit</extends>
   <header>XMatLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>XMatComboBox</class>
   <extends>QComboBox</extends>
   <header>XMatComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>XMatLabel</class>
   <extends>QLabel</extends>
   <header>XMatLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
